src/stack.py

```diff
@@ -0,0 +1,26 @@
+"""implments the stack"""
+from linked_list import LinkedList
+
+
+class Stack(object):
+    """Is the start of the stack object class."""
+
+    def __init__(self, itter=None):
+        """Creates the new stack using the linked list."""
```

Comment:
Why is there a mention of the linked list in the doc string? It doesn't matter what you use to create the functionality you want. All that matters is that it works in the way that's expected.